src/engine/enginebuffer.cpp

```diff
@@ -83,6 +83,7 @@ EngineBuffer::EngineBuffer(const QString& group,
           m_dSlipRate(1.0),
           m_bSlipEnabledProcessing(false),
           m_slipModeState(SlipModeState::Disabled),
+          m_quantize(ControlFlag::AllowMissingOrInvalid),
```

Comment:
move delayed init from below to here

```suggestion
          m_quantize(ConfigKey(group, QStringLiteral("quantize"))),
```

Reply:
This was my [first attempt](https://github.com/mixxxdj/mixxx/commit/5b30f03873d21ec976a097a3809e14e402b20bfe#diff-c960dda3b73ecd4b1b7801a42ddf7c5b9f6427af1e83c23368dffbddb06ca391R86), but this [broke the tests](https://github.com/mixxxdj/mixxx/actions/runs/17083734462/job/48443507428)

My understanding of the situation is that this CO isn't not available at that time, and only becomes ready after its init [here](https://github.com/mixxxdj/mixxx/pull/15222/files/4e4e76b9c92fd7f35179af2c9bccfecb0f473f9a#diff-c960dda3b73ecd4b1b7801a42ddf7c5b9f6427af1e83c23368dffbddb06ca391R188)

Reply:
> My understanding of the situation is that this CO isn't not available at that time, and only becomes ready after its init [here](https://github.com/mixxxdj/mixxx/pull/15222/files/4e4e76b9c92fd7f35179af2c9bccfecb0f473f9a#diff-c960dda3b73ecd4b1b7801a42ddf7c5b9f6427af1e83c23368dffbddb06ca391R188)

Ah I see. Makes total sense. I guess this is the best reasonable thing to do here then.

> Okay as fix for the merge commit, then --in the another commit-- make QuantizeControl a member, too, (even though unneeded) and create all EngineControls in initializer list?
> Is this worth it?

Yes IMO, though not as part of a merge PR.